src/patternfly/components/Expandable/expandable.scss

```diff
@@ -10,12 +10,14 @@
   --pf-c-expandable__toggle--active--Color: var(--pf-global--link--Color--hover);
   --pf-c-expandable__toggle--focus--Color: var(--pf-global--link--Color--hover);
   --pf-c-expandable__toggle--m-expanded--Color: var(--pf-global--link--Color--hover);
-  --pf-c-expandable__toggle-icon--Transition: .2s ease-in 0s;
-  --pf-c-expandable--m-expanded__toggle-icon--Transform: rotate(90deg);
 
   // Expandable toggle icon
-  --pf-c-expandable__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
   --pf-c-expandable__toggle-icon--Color: var(--pf-global--Color--100);
+  --pf-c-expandable__toggle-icon--Transition: .2s ease-in 0s;
+  --pf-c-expandable--m-expanded__toggle-icon--Transform: rotate(90deg);
+
+  // Expandable toggle text
+  --pf-c-expandable__toggle-text--MarginLeft: calc(var(--pf-global--spacer--xs) + var(--pf-global--spacer--sm));
```

Comment:
Hmm... I'm curious if this is to account for the way the font awesome icon is drawn (tall/narrow), then rotated (short/wide) - and when it's rotated, it's done in place without moving anything around it, so it appears close to the text?

Assuming so, the react icon doesn't do that, so I wonder if we're going to create too much space there. Maybe we could use `min-width: 1em` on `.pf-c-expandable__toggle-icon` instead? That should scale with the icon font size. WDYT?

Reply:
This was to account for the space created between the icon and text that wasn't created by anything:

<img width="376" alt="Screen Shot 2020-04-07 at 9 56 33 AM" src="https://user-images.githubusercontent.com/20118816/78677851-37934200-78b6-11ea-9fc7-7cfd1341ce8e.png">

so when I added in the text class, the space went away, so I had to add a larger margin to account for that space missing. The spacing between icon and text are now the same:
<img width="208" alt="Screen Shot 2020-04-07 at 9 56 05 AM" src="https://user-images.githubusercontent.com/20118816/78678059-76c19300-78b6-11ea-8568-89145c9d7fa7.png">
<img width="217" alt="Screen Shot 2020-04-07 at 9 56 09 AM" src="https://user-images.githubusercontent.com/20118816/78678061-77f2c000-78b6-11ea-9543-69b6643ef328.png">